packages/components/src/components/table/component.tsx

```diff
@@ -120,11 +123,13 @@ export class KolTable implements Generic.Element.ComponentApi<RequiredProps, Opt
 				if (typeof value === 'undefined') {
 					value = [];
 				}
-				try {
-					value = parseJson<KoliBriDataType[]>(value);
-					// eslint-disable-next-line no-empty
-				} catch (e) {
-					// value behält den ursprünglichen Wert
+				if (typeof value === 'string' && ['[', '{'].includes(value[0])) {
```

Comment:
Hi @Wolfram-Zell,

das passiert ja implizit in der parse-Methode. Brauchen wir das sonst noch?